pcdsdevices/attenuator.py

```diff
@@ -902,6 +902,41 @@ def format_status_info(self, status_info):
 '''
 
 
+class AttenuatorCalculatorSXR_TwoBlade(AttenuatorCalculatorBase):
+    """
+    2 blade x 8 filter solid attenuator variant from the L2SI project.
+
+    Parameters
+    ----------
+    prefix : str
+        Full Solid Attenuator base PV.
+
+    name : str
+        Alias for the Solid Attenuator.
+    """
+
+    tab_component_names = True
+    first_filter = 1
+    num_filters = 4
```

Comment:
```suggestion
    num_filters = 2
```

This may be vestigial, but probably best to keep it correct regardless

Reply:
Note for myself: this is only used in the 18 blade att to help construct a mapping, and then is set to the new correct value elsewhere to maintain interfaces despite not being used again.

Reply:
Yes, I thought this was weird, but I kept it there since the AT2K2 also has it